stable_plugins/quantum_ml/qiskit_ml/variational_quantum_classifier/tasks.py

```diff
@@ -283,7 +313,13 @@ def calculation_task(self, db_id: int) -> str:
         label_to_int=label_to_int,
     )
 
-    info_str = f"_feature_map_{str(feature_map_enum.name).replace('_feature_map', '')}_feature_entanglement_{entanglement_pattern_feature_map}_ansatz_{vqc_ansatz_enum.name}_ansatz_entanglement_{entanglement_pattern_ansatz}"
+    concat_filenames = retrieve_filename_from_url(train_data_url)
+    concat_filenames += retrieve_filename_from_url(train_labels_url)
+    concat_filenames += retrieve_filename_from_url(test_data_url)
+    concat_filenames += retrieve_filename_from_url(test_labels_url)
+    filename_hash = get_readable_hash(concat_filenames)
+
+    info_str = f"_feature_map_{str(feature_map_enum.name).replace('_feature_map', '')}_feature_entanglement_{entanglement_pattern_feature_map}_ansatz_{vqc_ansatz_enum.name}_ansatz_entanglement_{entanglement_pattern_ansatz}_from_{filename_hash}"
```

Comment:
this looks already long enough to maybe use abbreviations

```suggestion
    info_str = f"_feature_map_{str(feature_map_enum.name).replace('_feature_map', '')}_e_{entanglement_pattern_feature_map}_ansatz_{vqc_ansatz_enum.name}_e_{entanglement_pattern_ansatz}_from_{filename_hash}"
```

Reply:
Should I also use abbreviations with other plugin outputs?

Reply:
Only if the filenames seem unreasonably long